submissions/7SergSmith7/Tiny JS World/index.js

```diff
@@ -0,0 +1,125 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Habitant {
+  constructor(species, name, gender, phrase, friends = [], legs, hands) {
+    this.species = species;
+    this.legs = legs;
+    this.hands = hands;
+    this.name = name;
+    this.gender = gender;
+    this.phrase = phrase;
+    this.friends = friends;
+    //this.addFriends(friends);
+  }
+
+  addFriends(newFriend) {
+    this.friends.push(newFriend);
+  }
```

Comment:
How about making this method so it could accept both a single `friend` and an array of `friends` as its parameter?
This way you really could call it from the constructor for consistency. Yet you'd need to initialize the property with an empty array, again for consistency.

Hints:
`Array.isArray()`
`typeof`

Reply:
done